src/test/java/core/basesyntax/service/calculation/BalanceCalculatorTest.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.calculation;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceCalculatorTest {
+    private static TransactionCalculation transactionCalculation;
+    private static FruitTransaction fruitTransaction;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        transactionCalculation = new BalanceCalculator();
+    }
+
+    @Test
+    public void balanceCalculator_Ok() {
```

Comment:
names of test methods should be like this <methodUnderTest>_<state>_<expectedBehavior>, in this case it could be `calculate_validTransaction_ok`, the same for other test methods in all test classes